x-pack/filebeat/docs/inputs/input-cel.asciidoc

```diff
@@ -647,6 +647,25 @@ This specifies SSL/TLS configuration. If the ssl section is missing, the host's
 CAs are used for HTTPS connections. See <<configuration-ssl>> for more
 information.
 
+[float]
+==== `resource.proxy_url`
+
+This specifies proxy configuration in the form of `http[s]://<user>:<password>@<server name/ip>:<port>`. Proxy headers may be configured using the `resource.proxy_headers` field which accepts a set of key/value pairs.
```

Comment:
Regarding `resource.proxy_headers`, I think should have dedicated entry on the page for that option too.

IIUC, it is a map that specifics additional headers that are sent to the proxy during CONNECT requests.

Reply:
I looked and I don't see it mentioned in other docs. Note that this change is a backport, so if we want to add that, I think it should be separate.